src/main/java/com/softserve/academy/spaced/repetition/config/EventListenerBean.java

```diff
@@ -38,6 +38,13 @@ public void handleCheckPointsEvent(ContextRefreshedEvent event) {
             }
         }
     }
+
+    @EventListener
+    public User updatePointsBalance(PointsBalanceEvent pointBalanceEvent) {
+        LOG.info("Updating points balance for {} {}", pointBalanceEvent.getUser().getPerson().getFirstName(),
+                pointBalanceEvent.getUser().getPerson().getLastName());
```

Comment:
A possible NullPointerException. I would write 
`LOG.info("Updating points balance for {}.", pointBalanceEvent.getUser())`
See my comment about making User required.